src/Sylius/Bundle/CoreBundle/Fixture/Factory/ChannelExampleFactory.php

```diff
@@ -68,6 +72,7 @@ public function __construct(
         $this->currencyRepository = $currencyRepository;
         $this->zoneRepository = $zoneRepository;
         $this->taxonRepository = $taxonRepository;
+        $this->shopBillingDataFactory = $shopBillingDataFactory;
```

Comment:
We need to add triggering deprecation error, as done [in line 67](https://github.com/Sylius/Sylius/pull/12027/files#diff-60c0dfd90ee70289d4ea18aca5f01bdbacdb5dae3add1e631369181e36db72bfR67) 👌